javav2/example_code/entityresolution/src/main/java/com/example/entity/scenario/EntityResScenario.java

```diff
@@ -0,0 +1,481 @@
+            }
+            return;
+        }
+        waitForInputToContinue(scanner);
+        logger.info(DASHES);
+
+        logger.info(DASHES);
+        logger.info("6. List Schema Mappings.");
+        try {
+            actions.ListSchemaMappings();
+        } catch (CompletionException ce) {
+            logger.error("Error retrieving schema mappings: " + ce.getCause().getMessage());
+            return;
+        }
+        waitForInputToContinue(scanner);
+        logger.info(DASHES);
+
+        logger.info(DASHES);
+        logger.info("7. Tag the {} resource.", jsonSchemaMappingName);
+        logger.info("""
+            Tags can help you organize and categorize your Entity Resolution resources. 
+            You can also use them to scope user permissions by granting a user permission 
+            to access or change only resources with certain tag values. 
+            In Entity Resolution, SchemaMapping and MatchingWorkflow can be tagged. For this example, 
+            the SchemaMapping is tagged.
+                """);
+        try {
+            actions.tagEntityResource(jsonSchemaMappingArn).join();
+        } catch (CompletionException ce) {
+            logger.error("Error tagging the resource: " + ce.getCause().getMessage());
+            return;
+        }
+
+        waitForInputToContinue(scanner);
+        logger.info(DASHES);
+
+        logger.info(DASHES);
+        logger.info("8. View the results of the AWS Entity Resolution Workflow.");
+        logger.info("""
+            You cannot view the result of the workflow that is in a running state.  
+            In order to view the results, you need to wait for the workflow that we started in step 3 to complete.
+                        
+            If you choose not to wait, you cannot view the results or delete the workflow. You would have to 
+            perform both tasks manually in the AWS Management Console.
+                       
+            This can take up to 30 mins (y/n).
+            """);
+        String viewAns = scanner.nextLine().trim();
+        if (viewAns.equalsIgnoreCase("y")) {
+            logger.info("You selected to view the Entity Resolution Workflow results.");
+            countdownWithWorkflowCheck(actions, 1800, jobId, workflowName);
+            try {
+                JobMetrics metrics = actions.getJobInfo(workflowName, jobId).join();
+                logger.info("Number of input records: {}", metrics.inputRecords());
+                logger.info("Number of match ids: {}", metrics.matchIDs());
+                logger.info("Number of records not processed: {}", metrics.recordsNotProcessed());
+                logger.info("Number of total records processed: {}", metrics.totalRecordsProcessed());
+                logger.info("""
+                      
+                      The output of the machinelearning-based matching job is a CSV file in the S3 bucket. The following is a sample of the output:
```

Comment:
Where is the csv file that was produced? We should show the real output instead of fake output. I would want to see where the output was retrieved from the S3 bucket and some of the actual data.

Reply:
Read results of the job are shown using this code 

 JobMetrics metrics = actions.getJobInfo(workflowName, jobId).join(); 

We also told the user the S3 bucket the CSV file is located in.